src/cli/imports.rs

```diff
@@ -213,33 +141,337 @@ impl fmt::Display for Import {
+            col_width[idx] = cmp::max(col_width[idx], print_str.len());
+        }
+
+        imports.push(print_vec);
+    }
+
+    // print collected imports
+    println!("Imports:");
+    print_imports(&headers, imports, &col_width, no_color);
+}
+
+fn print_imports(
+    headers: &Vec<&str>,
+    mut imports: Vec<Vec<String>>,
+    col_width: &Vec<usize>,
+    no_color: bool,
+) {
+    imports.sort();
+
+    print!("\t");
+    for (idx, hdr) in headers.iter().enumerate() {
+        let width = col_width[idx];
+        print!("{:width$}  ", hdr);
+    }
+    println!();
+
+    for import in imports {
+        print!("\t");
+        for (idx, print_str) in import.iter().enumerate() {
+            let width = col_width[idx];
+
+            if idx == 0 && !no_color {
+                print!("{:width$}  ", print_str.yellow());
+            } else if idx == 1 && !no_color {
+                print!("{:width$}  ", print_str.green());
+            } else if idx == 2 && !no_color {
+                print!("{:width$}  ", print_str.red());
+            } else if idx == 4 && !no_color {
+                print!("{:width$}  ", print_str.cyan());
+            } else {
+                print!("{:width$}  ", print_str);
+            }
+        }
+        println!();
+    }
+}
+
+// ELF Helper Functions -----------------------------------------------------------------------------------------------
+
+fn get_elf_symbol_version_string(elf: &goblin::elf::Elf, sym_idx: usize) -> Option<String> {
+    let versym = elf.versym.as_ref()?.get_at(sym_idx)?;
+
+    if versym.is_local() {
+        return Some("local".to_string());
+    } else if versym.is_global() {
+        return Some("global".to_string());
+    } else if let Some(needed) = elf
+        .verneed
+        .as_ref()?
+        .iter()
+        .find(|v| v.iter().any(|f| f.vna_other == versym.version()))
+    {
+        if let Some(version) = needed.iter().find(|f| f.vna_other == versym.version()) {
+            let need_str = elf.dynstrtab.get_at(needed.vn_file)?;
+            let vers_str = elf.dynstrtab.get_at(version.vna_name)?;
+            return Some(format!("{}, {}", need_str, vers_str));
+        }
+    }
+
+    None
+}
+
+fn get_plt_address(elf: &goblin::elf::Elf, reloc: &goblin::elf::Reloc) -> Option<u64> {
+    // only handle JUMP_SLOT relocations
+    if reloc.r_type == goblin::elf::reloc::R_X86_64_JUMP_SLOT {
+        let reloc_idx = elf.pltrelocs.iter().position(|r| r == *reloc)?;
+        let plt_stub_len = 16;
+        let offset = (reloc_idx as u64 + 1) * plt_stub_len;
+
+        let plt_base = &elf
+            .section_headers
+            .iter()
+            .find(|s| {
+                elf.shdr_strtab
+                    .get_at(s.sh_name)
+                    .unwrap_or("err")
+                    .eq(".plt")
+            })?
+            .sh_addr;
+
+        return Some(plt_base + offset);
+    }
+
+    None
+}
+
+fn get_symbol_r_type_64(sym_type: u32) -> String {
```

Comment:
Thanks for adding these lookup tables!

Was wondering if this sort of pretty printing should be a feature of `goblin` itself, but these are actually constants that already implement [`Display`](https://doc.rust-lang.org/std/fmt/trait.Display.html) to print the number itself (not helpful for us).